src/feasibility_form_nls.jl

```diff
@@ -290,6 +290,47 @@ function hprod!(nlp :: FeasibilityFormNLS, xr :: AbstractVector, y :: AbstractVe
   return hv
 end
 
+function jth_hess_coord!(nlp :: FeasibilityFormNLS, x :: AbstractVector{T}, j :: Integer, vals :: AbstractVector{T}) where T
+  @lencheck nlp.meta.nvar x
+  @assert 1 ≤ j ≤ nlp.meta.ncon
+  @lencheck nlp.meta.nnzh vals #nlp.meta.nnzh is of length nnzhF+nnzhc+ne
+  increment!(nlp, :neval_jhess)
+  n, m, ne = nlp.internal.meta.nvar, nlp.internal.meta.ncon, nlp.internal.nls_meta.nequ
+  nnzhF = nlp.internal.nls_meta.nnzh
+  nnzhc = m > 0 ? nlp.internal.meta.nnzh : 0
+  IF = 1:nnzhF
+  Ic = nnzhF+1:nnzhF+nnzhc
+  if 1 ≤ j ≤ ne
+    yj = zeros(T, ne)
+    yj[j] = T(1)
```

Comment:
```suggestion
    yj[j] = one(T)
```
I also think it's safe to say `yj[j] = 1` because it will convert the integer to the appropriate float. I don't know if there's any difference in performance.